### ISSUE TITLE
Past conference programms

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding.github.io#723]
After a conference is over it is no longer possible to see the detailed program in conftool, e.g., <https://www.conftool.net/music-encoding2022/sessions.php>. It would be good if those are preserved somehow and made accessible. 

The publication from the MEC 2022 only contains the articles and not the program.

**Comment by musicEnfanthen** [comment]
We have the program for the conferences on the pages, like https://music-encoding.org/conference/2024/program/ or https://music-encoding.org/conference/2022/program/

Is there something more you would need?

**Comment by rettinghaus** [comment]
I can only see the time slots:

> 10:30-12:30 Paper presentations (Session 1)
 13:30-14:00 Poster slam 

But also the titles and presenters would be nice.

**Comment by bwbohl** [comment]
![Screenshot 2025-01-07 at 21 50 09](https://github.com/user-attachments/assets/48405984-eed9-4c72-a0ce-5eea1143ace8)
![Screenshot 2025-01-07 at 21 49 44](https://github.com/user-attachments/assets/7c585a91-6576-473e-afed-a393646524b3)

**Comment by rettinghaus** [comment]
![image](https://github.com/user-attachments/assets/dbb0a8a3-5d12-44ea-aa7d-f7bbcf2c7fc2)

**Comment by bwbohl** [comment]
@musicog as you were PC chair for 2022 do you still have those details somewhere?

**Comment by ahankinson** [comment]
Are the full proceedings for each year linked anywhere? All I see on the "Proceedings" pages are a list of (some of) the papers for some of the conferences (2022 and 2023 is missing). Where do I find the past proceedings?

**Comment by musicEnfanthen** [comment]
Yes, the proceedings page is the place. For 2022 and 2025 we are still missing the bibtex files (pinging @musicog and @annplaksin ). For 2023 there is a separate proceedings publication with jTEI. (Inclusion via bibtex would be great here too).

**Comment by bwbohl** [comment]
ping
